app/assets/javascripts/initializers/initializeUserProfilePage.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
 function initializeUserProfilePage() {
-  if (document.getElementById('user-profile-dropdown')) {
-    document.getElementById('user-profile-dropdown').onclick = () => {
-      document
-        .getElementById('user-profile-dropdownmenu')
-        .classList.toggle('showing');
-    };
+  if(document.querySelector('.profile-dropdown')){
+    var profileDropdownDiv = document.querySelector('.profile-dropdown');
+    var currentUser = userData();
+    var profileUser = JSON.parse(profileDropdownDiv.getAttribute('data-info'));
+
+    if(currentUser.username == profileUser.username){
+      profileDropdownDiv.hidden = true;
+    }
+    else{
```

Comment:
keep an eye on the spaces, and also eslint on my local machine is complaining about the `==` instead of `===`

Reply:
@rhymes What do you mean by spaces? whitespaces? or indentation?

Reply:
sorry, spaces between braces, like here for example:

`if(currentUser.username == profileUser.username){`

it should be 

`if (currentUser.username === profileUser.username) {`

thanks!

Reply:
Done 🎉